### PR TITLE
Add new fields to residents get endpoint

### PR DESCRIPTION
**Pull request by moarpheus**
## Link to JIRA ticket

[JIRA ticket](https://hackney.atlassian.net/browse/SCT-1693).

### *What is the problem we're trying to solve*

Adding [new fields](https://app.swaggerhub.com/apis/Hackney/social-care-case-viewer-api/1.0.0#/Residents%2C%20FOR%20DISCUSSION%20ONLY/get_residents__id_) to the resident db table and GET endpoint

### *What changes have we introduced*

Expanded resident class to include all the new fields + classes for complex fields.

#### _Checklist_

- [ ] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [x] Added tests to cover all new production code
- [ ] Added comments to the README or updated relevant documentation _(add link to documentation)_, where necessary.
- [x] Checked all code for possible refactoring
- [x] Code pipeline builds correctly

### *Follow up actions after merging PR*

Need to run the migration against the staging DB

### CODE REVIEW

**Comment by jhackett1** [comment]
will this auto-generate a new swagger definition?

if not, can we double-check https://app.swaggerhub.com/apis/Hackney/social-care-case-viewer-api/1.0.0#/Residents%2C%20FOR%20DISCUSSION%20ONLY/get_residents__id_ is a correct description of these extra fields?

File: SocialCareCaseViewerApi/V1/Factories/EntityFactory.cs [hunk 1]
```diff
@@ -33,6 +43,27 @@ public static Address ToDomain(this DbAddress address)
             };
         }
 
+        public static LastUpdatedDomain DbLastUpdatedToDomain(LastUpdated lastUpdated)
+        {
+            return new LastUpdatedDomain()
+            {
+                Housing = lastUpdated.Housing,
+                ContactDetails = lastUpdated.ContactDetails
```
Comment (jhackett1) [inline SocialCareCaseViewerApi/V1/Factories/EntityFactory.cs:51]: these are arbitrary values for the sake of the tests, yes?

we just want this to be object with arbitrary keys and datetime values

File: SocialCareCaseViewerApi/V1/Infrastructure/Person.cs [hunk 2]
```diff
@@ -54,7 +65,32 @@ public class Person : IAuditEntity
 
         [Column("first_language")]
         [MaxLength(100)]
-        public string FirstLanguage { get; set; }
+        public string? FirstLanguage { get; set; }
+
+        [Column("fluent_in_english")]
```
Comment (jhackett1) [inline SocialCareCaseViewerApi/V1/Infrastructure/Person.cs:70]: double-checking: are these new keys camel-case or snake-case in the api response?

**Review by jhackett1 — APPROVED** [review]
two minor questions

File: SocialCareCaseViewerApi/V1/Factories/EntityFactory.cs (hunk 1 above)
Comment (moarpheus) [reply SocialCareCaseViewerApi/V1/Factories/EntityFactory.cs:51]: No these are actual values. Thanks for clarifying - I will amend.

**Pull request merged**